### ISSUE TITLE
Question: The val_loss about fine-tuning with Bridge V2 is a fixed value

### ISSUE THREAD

**Issue by XXXUAN0904** [Robot-VLAs/RoboVLMs#31]
When I used bridge v2 finetune RoboVLMs (FCDecoder), I set up training 400k iterations to verify every 100k, but I found that the val_loss did not change at all, and the effect of reasoning on simpler was that the robotic arm output the same action no matter where the object was, and I felt that I didn't learn anything, why is this? Is there a problem with the github code?

![Image](https://github.com/user-attachments/assets/42e3e9d4-c422-4fb5-917f-3327d6a9530b)
![Image](https://github.com/user-attachments/assets/18b8d56b-8bf1-418e-a523-4287b508c790)

**Comment by lixinghang12** [comment]
Could you share the training config and hyper parameters like learning rate and batch size? The training loss is quite high and unstable, and the validation loss is also very large. Maybe you can check if the val_gripper_acc is obvisouly higher than 50%.

**Comment by lixinghang12** [comment]
<img width="1369" alt="Image" src="https://github.com/user-attachments/assets/ba8ee6ae-3a4d-4a2f-b503-10386d3ca1b2" />
This is a training log from Bridge V2 with paligemma FC decoder.
